src/run/headers.ts

```diff
@@ -231,6 +232,12 @@ export const setCacheControlHeaders = (
     return
   }
 
+  if (status === 404 && request.url.endsWith('.php')) {
+    // temporary CDN Cache Control handling for bot probes
```

Comment:
It might be worth linking to a linear ticket here to help prompt a resolution around this block. Alternatively, we could also remove "temporary" from this description.